packages/nectar/src/sagas/messages/verifyMessage/verifyMessages.saga.ts

```diff
@@ -25,9 +25,7 @@ function* verifyMessage(message: ChannelMessage, crypto: SubtleCrypto): Generato
   const publicKeysMapping = yield* select(messagesSelectors.publicKeysMapping)
 
   let cryptoKey = publicKeysMapping[message.pubKey]
-  /* Sometimes an empty object is being stored as a key (async call problem?)
-     checking its content (key type) prevents invalid object from being passed to verification method */
-  if (!cryptoKey || !cryptoKey.type) {
+  if (cryptoKey === undefined) {
```

Comment:
Are there any cases where it isn't undefined but it's invalid in some other way? What will happen if so?

Reply:
That was the original source of the problem. Redux persist store was not able to correctly serialize/deserialize CryptoKey mapping which resulted in invalid data being present in the store - but it was incorrect in a different ways depending on the platform (web, mobile). That's why I added custom transformations for this type of data which simply ignores this mapping when flushing store state by setting it as undefined